project/AppDependencies.scala

```diff
@@ -19,7 +19,7 @@ import sbt._
 
 object AppDependencies {
 
-  private val bootstrapPlay28Version = "6.4.0"
+  private val bootstrapPlay28Version = "7.0.0"
```

Comment:
Why was this updated?

Reply:
It was updated as it was out of sync with the app dependencies before the revert

Reply:
OK